### DISCUSSION TITLE
App: Collection view gets stuck at "Loading..."

### DISCUSSION THREAD

**Discussion by obiwan87** [directus/directus#5857] [Archive]
Affected component: App

In the app, when I open the collections I get stuck at "Loading ..." and my data
won't be displayed.

What I did before: I removed a many-to-many relationship and added an 1-to-many relationship.

What I tried:
- restart docker
- hard reload (+ emptying cache)
- clean cookies 

Nothing helped.

---

### Project details
Directus Version: 9.0.0-rc.24
Environment: production
OS: Linux 4.15.0
Node: 14.15.2

**Comment by benhaynes** [comment]
https://docs.directus.io/getting-started/support/#troubleshooting-steps
